src/sage/interfaces/maxima.py

```diff
@@ -72,17 +72,17 @@
 ::
 
     sage: repr(F)
-    '-(y-x)*(y^4+x*y^3+x^2*y^2+x^3*y+x^4)'
+    '-...(y-x)*(y^4+x*y^3+x^2*y^2+x^3*y+x^4)...'
     sage: F.str()
```

Comment:
There's no reason to continue using the (complicated) expression `F` here. Something predictable like `maxime(1/2)` should work fine.